.github/workflows/ci_build_tests.yml

```diff
@@ -26,7 +26,8 @@ jobs:
         python -m pip install --upgrade pip
         # python -m pip install flake8 
         python -m pip install pytest cython pytest-cov
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        
+        if [ -f requirements.txt ]; then pip install -r requirements.txt --use-deprecated=legacy-resolver; fi
```

Comment:
after upgrading to new pip 20.3, looks like they have a new legacy resolver. It's a dumpster fire, so we're using this flag temporarily to use the old resolver.